app/controllers/application_controller.rb

```diff
@@ -214,9 +216,13 @@ def render_exception(msg)
         end
       end
       format.html do # HTML, send error screen
-        @layout = "exception"
-        response.status = 500
-        render(:template => "layouts/exception", :locals => {:message => msg})
+        if event.instance_of?(MiqException::RbacPrivilegeException)
```

Comment:
We probably want to switch to a case statement like above in case we add other exception classes... otherwise we'll have more if statements.


```ruby
case exception
when ::MiqException::RbacPrivilegeException
  redirect_to(:controller => 'dashboard', :action => "auth_error")
else
  @layout = "exception"
  response.status = 500
  render(:template => "layouts/exception", :locals => {:message => msg})
end